.github/workflows/build-container.yaml

```diff
@@ -54,38 +49,42 @@ jobs:
           username: ${{ github.actor }}
           password: ${{ secrets.GITHUB_TOKEN }}
       - id: meta
-        if: ${{ env.PUBLISH }}
         name: Extract metadata (tags, labels) for Docker
         uses: docker/metadata-action@902fa8ec7d6ecbf8d84d538b9b233a880e428804 #v5.7.0
         with:
           images: ${{ env.REGISTRY }}/${{ env.IMAGE_NAME }}
+          tags: |
+            type=ref,event=tag
+            type=ref,event=pr
+            type=ref,event=branch
```

Comment:
See https://github.com/docker/metadata-action?tab=readme-ov-file#typeref

Reply:
> See https://github.com/docker/metadata-action?tab=readme-ov-file#typeref

Even after reading the metadata-action docs, I don't really understand what's going on here. Are we being explicit now because we don't `type=schedule`, which is part of the default?

Reply:
I'm partially being explicit for the sake of being explicit.